docker-compose.yml

```diff
@@ -7,8 +7,8 @@ services:
     build:
       context: .
       dockerfile: UT4MasterServer/Dockerfile
+    hostname: api.ut4uu.com
     ports:
-      - "80:80"
```

Comment:
We have Swagger here. We can't disable port 80

Reply:
Swagger serves fine on 443

Reply:
But the web front end should also be served https, just haven't gotten to that yet